### ISSUE TITLE
Hand-implement ResNet 

### ISSUE THREAD

**Issue by jiwoncpark** [jiwoncpark/h0rton#24]
Hand-implementing ResNet instead of calling torchvision's will enable us to:
- modify of the layers (dropout, concrete dropout)
- remove fluff (like pretrained options)
- save GPU memory since we can bypass concatenating 3 channels
- save training time since we can bypass normalization of the 3 channels

**Comment by jiwoncpark** [comment]
Done and closing.
